### PR TITLE
Add openSUSE Leap 42.3 support

### PR DESCRIPTION
**Pull request by hwoarang**
This PR adds support for openSUSE Leap 42.3 support. The Ceph packages can either be installed from the official repositories or from the OBS one.

### CODE REVIEW

**Comment by guits** [comment]
@leseb @andrewschoen @alfredodeza 
any thoughts on this PR?

**Comment by hwoarang** [comment]
Does anyone know what needs to be done to add a Jenkins job for openSUSE as well so we can test it in the same way as the rest of the distros?

**Comment by alfredodeza** [comment]
we don't have the ability to test openSUSE as it is not even part of the Openstack provider we use to launch ephemeral builders. If we did, we would need to change a bunch of services to support new configuration

File: roles/ceph-docker-common/tasks/misc/ntp_suse.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+- name: include ../checks/check_ntp_suse.yml
```
Comment (leseb) [inline roles/ceph-docker-common/tasks/misc/ntp_suse.yml]: Where is  `check_ntp_suse.yml`?
Also, can you merge this with `ntp_redhat.yml`?

**Comment by hwoarang** [comment]
@leseb I think it should be OK now
@alfredodeza oh that's a shame. However, we can still test openSUSE + Ceph in the OpenStack Ansible project so we can capture potential future failures.

**Pull request merged**